certificate-service/app/src/main/java/se/inera/intyg/certificateservice/application/certificate/service/converter/CertificateMetadataConverter.java

```diff
@@ -114,6 +114,9 @@ public CertificateMetadataDTO convert(Certificate certificate) {
                 .toList()
                 : null
         )
+        .responsibleHospName(certificate.certificateMetaData().responsibleHospName() != null
```

Comment:
Do we want to use "responsibleHospName" or name it something different? I remember we removed the term hosp when working with intyg-proxy.